main/common/util/filereader_utils.py

```diff
@@ -58,36 +60,37 @@ def findstars(path, saturation, subframe=None, return_data=False):
     if not subframe:
         starfound = photutils.find_peaks(data, threshold=threshold, box_size=50, border_width=500,
                                          centroid_func=photutils.centroids.centroid_com)
+        n = 0
+        stars = []
+        peaks = []
+        for _ in starfound:
+            bad_pixel = False
+            x_cent = starfound['x_peak'][n]
+            y_cent = starfound['y_peak'][n]
+            peak = image[y_cent, x_cent]
+            if peak >= (saturation * 2) ** 2:
+                bad_pixel = True
+            pixels = [(y_cent, x_cent + 1), (y_cent, x_cent - 1), (y_cent + 1, x_cent), (y_cent - 1, x_cent)]
+            for value in pixels:
+                if image[value[0], value[1]] < 1.2 * median:
+                    bad_pixel = True
+            if bad_pixel:
+                n += 1
```

Comment:
Not a big deal but n += 1 could be placed after line 69.  You also could continue at each of the saturation and pixel checks so as not continue to line 73 if the if clause on 71 is true.  This would allow you to get rid of the bad_pixel bool.  One less thing to track and a little cleaner.